### ISSUE TITLE
[Issue]: Xbox One controller not showing under game settings

### ISSUE THREAD

**Issue by Euroblitz** [an-anime-team/aagl-ebuilds#6]
### Distro

Gentoo Base System release 2.8 x86_64
KDE Plasma 5.24.6 X.org
5.15.63-gentoo-dist

### Installation Type

AppImage

### Describe the issue

Hello there,

I can't get my Xbox controller showing or working in Genshin Impact game while using Gentoo and the launcher (AppImage, didn't try the ebuild as I already had all of the dependencies installed), have been using the launcher for over a month on Debian and I was able to play the game with no problems using my controller, but when I tried to use it on Gentoo the game settings only shows the "Keyboard" options.

My controller (/dev/js0) shows output while I do `cat /dev/js0`, and I can see the input and configure it under Plasma's control panel "Game Controllers" option. I tried to mess with Xorg drivers and tried enabling `USE="udev"` and `INPUT_DEVICES="libinput joystick"` but got no success.

No useful logs are provided when trying to run the AppImage through the terminal, neither enabling "Use Terminal" option, but I can dump them if neccessary.

### Reproduction Steps

1. Go to game settings
2. Try to select "Joystick" under control option
3. Only option shown is "Keyboard"

**Comment by jiro-too** [comment]
Raise this issue in the main launcher page, this isn't the correct place.

**Comment by natimerry** [comment]
Well you are using dist-kernel so I am not sure to what extent I can help. Usually with a custom kernel you need to set `CONFIG_JOYSTICK_XPAD`. It is located in `Device Drivers -> Input device support -> Joysticks -> Xbox gamepad` in the kernel.

If the dist-kernel provides an config file (`/boot/config-5.19.6-stella`) for example can you try run `cat config-file | grep JOYSTICK_XPAD`

Just found this, you might wanna take a look: https://packages.gentoo.org/packages/games-util/xboxdrv

**Comment by Schievel1** [comment]
The config of the currently running kernel should be in /proc/config.gz

Although I don't think this option is disabled in the dist-kernel, but you can compile dist-kernel with your own config via savedconfig so you never know.

**Comment by Euroblitz** [comment]
Tried enabling `JOYSTICK_XPAD` and all gamepad related options in kernel config, still no success, the same thing applies to the `gentoo-kernel-bin` kernel. I've also installed the `anime-game-launcher` as an ebuild instead of using the AppImage, but no luck.

I've also tried to install xboxdrv package as I was told to do, but still no `Joystick` option shows under game settings. 

Current .config kernel file:
https://0x0.st/oO6P.txt
